### DISCUSSION TITLE
Can you have the oculus 3D default controllers visible?

### DISCUSSION THREAD

**Discussion by michaelcvoigt** [Rufus31415/Simple-WebXR-Unity#43] [Q&A]
Can you have the oculus 3D default controllers visible?

**Comment by Rufus31415** [comment]
That's possible if you use MRTK, but no examples are provided.
 MRTK is able to load 3D controller models. 

You have to use one of the MRTK + SimpleWebXR example scene and configure MRTK to load Oculus controllers :
https://docs.microsoft.com/fr-fr/windows/mixed-reality/mrtk-unity/supported-devices/controllers-mrtk?view=mrtkunity-2021-05#oculus-controllers
